Tasks/AzureMysqlDeploymentV1/sql/MysqlClient.ts

```diff
@@ -125,7 +125,7 @@ export class MysqlClient implements ISqlClient {
             fileSourceArgument = " -e" + '"' + this._azureMysqlTaskParameter.getSqlInline() + '"';
         }
         else {
-            fileSourceArgument = " -e" + '" source ' + packageUtility.PackageUtility.getPackagePath(this._azureMysqlTaskParameter.getSqlFile()) + '"';
+            fileSourceArgument = ` -e "source ${packageUtility.PackageUtility.getPackagePath(this._azureMysqlTaskParameter.getSqlFile())};"`;
```

Comment:
small: I meant to create a local var as below
```javascript
const sourcePath = packageUtility.PackageUtility.getPackagePath(this._azureMysqlTaskParameter.getSqlFile());
fileSourceArgument = ` -e "source ${sourcePath};"`;
```

Also it would be great if similar change is done in the corresponding if block as well.

Reply:
as this local variable will be used at one place only and whole string is added to another variable. I will keep this code as it is for now.